packages/web/src/brandkit/IconsExchangePage.tsx

```diff
@@ -0,0 +1,139 @@
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import CCLicense from 'src/brandkit/common/CCLicense'
+import { GAP } from 'src/brandkit/common/constants'
+import { brandStyles } from 'src/brandkit/common/constants'
+import Page, { ICONS_PATH } from 'src/brandkit/common/Page'
+import PageHeadline from 'src/brandkit/common/PageHeadline'
+import IconShowcase from 'src/brandkit/common/Showcase'
+import { AssetTypes, EXCHANGE_ICONS_PKG_TRACKING, trackDownload } from 'src/brandkit/tracking'
+import { NameSpaces, useTranslation } from 'src/i18n'
+import Button, { BTN } from 'src/shared/Button.3'
+import { hashNav } from 'src/shared/menu-items'
+import { colors } from 'src/styles'
+
+const icons = [
+  {
+    description: 'cUSD Exchange Icon, Full Color\n',
+    uri: '/images/marketplace-icons/icon-celo-dollar-color-f.svg',
+  },
+  {
+    description: 'cGLD Exchange Icon, Full Color\n',
+    uri: '/images/marketplace-icons/icon-celo-gold-color-f.svg',
+  },
+  null,
+  null,
+  {
+    description: 'cUSD Exchange Icon, Monochrome\n',
+    uri: '/images/marketplace-icons/icon-celo-dollar-black-f.svg',
+    variant: 'circle-white',
+  },
+  {
+    description: 'cGLD Exchange Icon, Monochrome\n',
+    uri: '/images/marketplace-icons/icon-celo-gold-black-f.svg',
+    variant: 'circle-white',
+  },
+  {
+    description: 'cUSD Exchange Icon, Reverse Monochrome\n',
+    uri: '/images/marketplace-icons/icon-celo-dollar-white-f.svg',
+    variant: 'circle-black',
+  },
+  {
+    description: 'cGLD Exchange Icon, Reverse Monochrome\n',
+    uri: '/images/marketplace-icons/icon-celo-gold-white-f.svg',
+    variant: 'circle-black',
+  },
+]
+
+const IconExchangePage: NextPage = React.memo(() => {
+  const { t } = useTranslation(NameSpaces.brand)
+  return (
+    <Page
+      title={t('exchangeIcons.title')}
+      metaDescription={t('exchangeIcons.headline')}
+      path={ICONS_PATH}
+      sections={[{ id: hashNav.brandIcons.overview, children: <Overview /> }]}
+    />
+  )
+})
+
+export default IconExchangePage
+
+export interface IconData {
+  description: string
+  name: string
+  preview: string
+  uri: string
+  tags: string[]
+  id: string
+}
+
+function Overview() {
+  const { t } = useTranslation(NameSpaces.brand)
+
+  const onPressDownload = React.useCallback(async () => {
+    await trackDownload(EXCHANGE_ICONS_PKG_TRACKING)
+  }, [])
+
+  return (
+    <View style={styles.container}>
+      <PageHeadline title={t('exchangeIcons.title')} headline={t('exchangeIcons.headline')} />
+
+      <Button
+        kind={BTN.PRIMARY}
+        text={t('logo.overviewBtn')}
+        onPress={onPressDownload}
+        href="/assets/CeloMarketplaceIcons.zip"
+      />
+
+      <CCLicense textI18nKey="exchangeIcons.license" />
+
+      <View style={styles.root}>
+        <View style={brandStyles.tiling}>
+          {icons.map((icon, i) =>
+            icon === null ? (
+              <View key={i} style={styles.empty} />
```

Comment:
I think i see what you are doing here, however on tablet sizes this causes gaps. 
![Screen Shot 2020-05-04 at 9 34 35 AM](https://user-images.githubusercontent.com/3814795/80990161-ced2b300-8dea-11ea-9234-127783dd49f8.png)

Maybe try having two icon arrays for color and monochrome and places them in their own `brandstyles.tiling` Views

Reply:
Ops! I didn't think about small sizes. Sure, I'll work on it. Another possible option is grouping it in couples in a non-wrapper component. So it is going to cut the couples.